ExampleModels/ForwardKinematics/test/runtests.jl

```diff
@@ -0,0 +1,21 @@
+using ForwardKinematics
+using Test
+
+floatEquals(x, y, eps) = abs(x - y) <= eps
+
+# Simple test
+point = forward_kinematics([pi], [1.0])
+@test floatEquals(point.x, -1, 0.00001)
+@test floatEquals(point.y,  0, 0.00001)
+
+# Exception due to unequal sizes
+try
```

Comment:
What is going on here?

Reply:
This is testing that we properly fail (throw an exception) if the argument have unequal sizes